met-web/src/components/engagement/form/AddSurveyBlock.tsx

```diff
@@ -81,6 +94,33 @@ export const AddSurveyBlock = () => {
         );
     };
 
+    const tabs = [
+        { Upcoming: SubmissionStatus[SubmissionStatus.Upcoming] },
+        { Open: SubmissionStatus[SubmissionStatus.Open] },
+        { Closed: SubmissionStatus[SubmissionStatus.Closed] },
+    ];
+    const [selectedTabName, setselectedTabName] = useState('');
+    const updateTabName = (name: string): void => {
+        setselectedTabName(name);
+    };
+
+    const richtexteditor = (Header: string, Paragraph: string) => {
+        return (
+            <RichTextEditor
+                initialRawEditorState={
```

Comment:
We shouldn't manually create an editor state like this we should update RichTextEditor to accept initialPlainText and follow this example 
![image](https://user-images.githubusercontent.com/91914654/208715140-164382af-e62c-4315-9d6a-2b121d626e49.png)

Reply:
Thanks used the suggested logic